### ISSUE TITLE
useExec throws with a JS error when the command fails.

### ISSUE THREAD

**Issue by cornedor** [raycast/extensions#2781]
Raycast version: @raycast/utils any version with useExec.

## Description

A incorrect error gets thrown when the useExec hook is used.

## Steps To Reproduce

1. Execute this snippet:

```
useExec('something that will throw an error');
```

2. The command will fail as expected, but failing throws the following error:

## The current behavior

```
/Users/cdorrestijn/GitHub/extensions/extensions/colima/node_modules/@raycast/utils/dist/useExec.js:210
    error.escapedCommand = escapedCommand;
                           ^

ReferenceError: escapedCommand is not defined
    at null.makeError (/Users/cdorrestijn/GitHub/extensions/extensions/colima/node_modules/@raycast/utils/dist/useExec.js:210:28)
    at Object.defaultParsing (/Users/cdorrestijn/GitHub/extensions/extensions/colima/node_modules/@raycast/utils/dist/useExec.js:226:31)
    at Object.current (/Users/cdorrestijn/GitHub/extensions/extensions/colima/node_modules/@raycast/utils/dist/useExec.js:271:31)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

```

## The expected behavior

The correct error is thrown. `escapedCommand` is defined, or is not set on the error.

**Comment by elisaado** [comment]
It seems that this error is also thrown for commands that complete successfully

**Comment by mathieudutour** [comment]
Oops, fixed the issue in 1.3.2
